plugins/auth-backend-module-oidc-provider/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@backstage/plugin-auth-backend-module-oidc-provider",
+  "description": "The oidc-provider backend module for the auth plugin.",
+  "version": "0.1.0-next.1",
```

```suggestion
  "version": "0.0.0",
```

Starting at this for new packages

---

Sure thing will make this change.